product_variant_supplierinfo/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+Product suppplier info per variant
+==================================
+
+This module limits supplier info to product variant scope, instead of having
+the same record for all variants.
+
+Installation
+============
+
+This module depend of purchase module
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/137/8.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
```

Comment:
The new template shortens this (https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst)